code/modules/mob/living/carbon/human/npc/dungeon_pack_npc.dm

```diff
@@ -0,0 +1,768 @@
+	if(prob(5))
+		target.apply_damage(rand(20, 40), BRUTE, BODY_ZONE_CHEST)
+		var/obj/item/bodypart/chest = target.get_bodypart(BODY_ZONE_CHEST)
+		if(chest)
+			var/datum/wound/W2 = new /datum/wound/fracture()
+			W2.apply_to_bodypart(chest)
+
+	else
+		target.apply_damage(rand(10, 20), BRUTE, BODY_ZONE_CHEST)
+
+	return TRUE
+
+
+/// ========== WRESTLER ==========
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/wrestler/Life()
+	. = ..()
+	if(world.time >= next_cast)
+		next_cast = world.time + 150
+		if(target && target.stat == CONSCIOUS && get_dist(src, target) <= 1)
+			say("You’re going down!")
+			target.Stun(50)
+			target.Knockdown(50)
+			spawn(100)
+				var/list/enemies = list()
+				for(var/mob/living/L in view(5, src))
+					if(L != src && L.stat == CONSCIOUS && disjoint_lists(L.faction, src.faction))
+						enemies += L
+				if(enemies.len)
+					target = pick(enemies)
+
+/// ========== ROGUE ==========
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/rogue/Life()
+	. = ..()
+	if(world.time >= next_cast)
+		next_cast = world.time + 150
+		var/list/enemies = list()
+		for(var/mob/living/L in view(7, src))
+			if(L != src && L.stat != DEAD && disjoint_lists(L.faction, src.faction))
+				enemies += L
+		if(enemies.len)
+			var/mob/living/target = pick(enemies)
+			say("Shadows guide me...")
+			step_towards(src, target)
+			target.visible_message(span_warning("[src] suddenly appears behind [target]!"))
+			spawn(20)
+				if(get_dist(src, target) <= 1)
+					src.visible_message(span_danger("[src] strikes from the shadows!"))
+					target.apply_damage(rand(20, 80), BRUTE)
+			spawn(100)
+				if(get_dist(src, target) <= 2)
+					say("Let’s slow you down.")
+					target.apply_damage(rand(10, 20), BRUTE, BODY_ZONE_L_LEG)
+					target.Slowdown(40)
+
+/// ========== MAGE ==========
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/mage/Life()
+	. = ..()
+	if(world.time >= next_cast)
+		next_cast = world.time + 180
+		var/list/enemies = list()
+		for(var/mob/living/L in view(8, src))
+			if(L != src && L.stat != DEAD && disjoint_lists(L.faction, src.faction))
+				enemies += L
+		if(enemies.len)
+			var/mob/living/target = pick(enemies)
+			say("Arcana, strike true!")
+			target.visible_message(span_warning("A bolt of arcane energy slams into [target]!"))
+			target.apply_damage(rand(30, 60), BURN)
+			if(prob(30))
+				explosion(get_turf(target), 0, 1)
+			spawn(100)
+				say("Feel your soul burn!")
+				target.apply_damage(rand(15, 45), BURN)
+				target.Slowdown(30)
+
+/// ========== SILENCER ==========
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/silencer/Life()
+	. = ..()
+	if(world.time >= next_cast)
+		next_cast = world.time + 300
+		say("Silence!")
+		for(var/mob/living/L in view(5, src))
+			if(L != src && L.stat != DEAD && disjoint_lists(L.faction, src.faction))
+				to_chat(L, span_warning("A wave of oppressive silence washes over you..."))
+				L.Stun(10)
+		spawn(100)
```

Comment:
```suggestion
		spawn(10 SECONDS)
```
also personally i would prefer making this a separate proc and using `addtimer()` for it. it's poor form to use spawn() or sleep() in `Life()`.